queries.py

```diff
@@ -26,11 +26,19 @@ def add_new_email(email,user_ssn,verified=1):
     return connect.exec(f"""INSERT INTO ELEC_ADDRESS VALUES('{email}','{verified}', 'EMAIL');\n
                             INSERT INTO EMAIL VALUES('{email}','{user_ssn}');\n""")
 
-def remove_email():
-    pass
-
-def add_new_phone():
-    pass
+def remove_email(email):
+    return connect.exec(f"""DELETE FROM ELEC_ADDRESS WHERE Identifier='{email}');\n
+                            DELETE FROM EMAIL WHERE Identifier='{email}');\n""")
+
```

Comment:
- This also needs to delete the rows in SEND_TRANSACTION and REQUEST_TRANSACTION that reference the deleted email.
- Minor syntax error: you have a single closed parantheses ) still at the end of line 30 and line 31